src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -107,8 +107,12 @@ function WorkspaceMembersPage(props) {
     useEffect(() => {
         validateSelection();
     }, [props.preferredLocale, validateSelection]);
-
+    const accountIDs = useMemo(() => _.keys(props.policyMembers), [props.policyMembers]);
```

Comment:
@mollfpr accountIDs is calculated from policyMembers so I put it in useMemo to avoid re-calculate it. But if you see it is unnecessary we can remove useMemo

Reply:
@DylanDylann I guess we can keep it. It make sense.